openbb_platform/providers/fmp/openbb_fmp/models/form_13f.py

```diff
@@ -0,0 +1,109 @@
+from openbb_core.provider.standard_models.form_13FHR import (
+    Form13FHRData,
+    Form13FHRQueryParams,
+)
+from openbb_core.provider.utils.errors import EmptyDataError
+from openbb_core.provider.utils.helpers import amake_request
+from openbb_fmp.utils.helpers import create_url
+from pydantic import Field
+
+
+class FMPForm13FHRQueryParams(Form13FHRQueryParams):
+    """Form 13f Query Parameters.
+
+    Source: https://financialmodelingprep.com/api/v3/form-thirteen/0001388838?date=2021-09-30
+    """
+
+
+class FMPForm13FHRData(Form13FHRData):
+    """Form13 FHR Data Model."""
+
+    __alias_dict__ = {
+        "period_ending": "date",
+        "issuer": "nameOfIssuer",
+        "principal_amount": "shares",
+        "asset_class": "titleOfClass",
+        "filling_date": "fillingDate",
+        "accepted_date": "acceptedDate",
+        "ticker_cusip": "tickercusip",
+        "final_link": "finalLink",
+    }
+    filling_date: dateType = Field(
+        default=None, description="Date when the filing was submitted to the SEC."
+    )
+    accepted_date: dateType = Field(
+        default=None, description="Date when the filing was accepted by the SEC."
+    )
+    ticker_cusip: Optional[str] = Field(
+        default=None, description="Ticker symbol associated with the CUSIP."
+    )
+    link: Optional[str] = Field(
+        default=None, description="URL link to the SEC filing on the SEC website."
+    )
+    final_link: Optional[str] = Field(
+        default=None,
+        description="URL link to the XML information table of the SEC filing.",
+    )
+
+
+class FMPForm13FHRFetcher(
+    Fetcher[
+        FMPForm13FHRQueryParams,
+        List[FMPForm13FHRData],
+    ]
+):
+    """Fetches and transforms data from the Form 13f endpoints."""
+
+    @staticmethod
+    def transform_query(params: Dict[str, Any]) -> FMPForm13FHRQueryParams:
+        """Transform the query params."""
+        return FMPForm13FHRQueryParams(**params)
+
+    @staticmethod
+    async def aextract_data(
+        query: FMPForm13FHRQueryParams,
+        credentials: Optional[Dict[str, str]] = None,
+        **kwargs: Any,
+    ) -> List[Dict]:
+        """Return the raw data from the House Disclosure endpoint."""
+        symbols = query.symbol.split(",")
+        results: List[Dict] = []
+
+        async def get_one(symbol):
+            """Get data for the given symbol."""
+            api_key = credentials.get("fmp_api_key") if credentials else ""
+            url = create_url(
+                3,
+                f"form-thirteen/{symbol}",
+                api_key,
+                query,
+                exclude=["symbol", "limit"],
+            )
+            result = await amake_request(url, **kwargs)
+            if not result or len(result) == 0:
+                warn(f"Symbol Error: No data found for symbol {symbol}")
+            if result:
+                results.extend(result)
+
+        await asyncio.gather(*[get_one(symbol) for symbol in symbols])
```

Comment:
As noted already, we are more interested in getting multiple filings from the same symbol, rather than the same date for multiple symbols. If we only allow 1 symbol, but multiple dates, we'll need to handle each date for the given symbol.